tests/e2e/e2e.sh

```diff
@@ -17,83 +17,57 @@
 set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
+ROOT=$(cd ../../../; pwd)
 
-function cleanup_kind_cluster() {
-    kind export logs --name istio-testing "${ARTIFACTS}/kind"
-    kind delete cluster --name=istio-testing
-}
-
-function setup_kind_cluster() {
-  # Delete any previous e2e KinD cluster
-  echo "Deleting previous KinD cluster with name=istio-testing"
```

Comment:
imo setup shouldn't delete, it should setup.  If we need to call delete to get the system progressing, thats fine.  i recommend not confusing the function names with what the functions do.